core/server/proxy/src/main/java/alluxio/proxy/s3/signature/SignerConstants.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.proxy.s3.signature;
+
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * constant key for sign.
+ */
+public final class SignerConstants {
+  public static final String IDENTIFIER = "AWS";
```

Comment:
If a constant is only used in one class, define it in that class and make it private.